e2e-tests/e2e-tests.ts

```diff
@@ -28,17 +28,25 @@ function delay (timeout) {
 }
 
 describe('A non-matching journey', () => {
-  it('should succeed', async () => {
-    await journey().then(async function () {
+  it('without eIDAS should succeed', async () => {
+    await eidasEnabledJourney().then(async function () {
```

Comment:
I think this would read more clearly if you wrote eidasEnabledJourney(false).

Reply:
done